src/main/java/uk/gov/hmcts/reform/professionalapi/domain/Organisation.java

```diff
@@ -87,7 +87,7 @@ public class Organisation implements Serializable {
     @Column(name = "LAST_UPDATED")
     private LocalDateTime lastUpdated;
 
-    @Column(name = "ORG_TYPE_KEY")
+    @Column(name = "ORG_TYPE")
     private String orgTypekey;
```

Comment:
is it better if we rename the variable as well? what do you think!

Reply:
We can do that but need to do again changes in other 3 files at 6 places.

Reply:
Will do the changes for that and push code shortly